Update bootprep and config overview (#36)
Cray-HPE/docs-uan: docs/portal/developer-portal/operations/Basic_UAN_Configuration.md

Configuration of UAN nodes is performed by the Configuration Framework Service \(CFS\). CFS can apply configuration to both images and nodes. When the configuration is applied to nodes, the nodes must be booted and accessible through SSH over the Node Management Network \(NMN\).

> The preferred method of creating CFS configurations is to use the Shasta Admin Toolkit
> (SAT) `sat bootprep` command.  This command automates the creation of IMS images, CFS
> configurations, and BOS session templates. See [Create UAN Boot Images](Create_UAN_Boot_Images.md) for more details.

Reviewer: ```suggestion
The preferred method of creating CFS configurations is to use the Shasta Admin Toolkit (SAT) `sat bootprep` command.  This command automates the creation of IMS images, CFS configurations, and BOS session templates. See [Create UAN Boot Images](Create_UAN_Boot_Images.md) for more details.
```
Taking out the newline

> CFS uses configuration layers. Configuration layers allow the sharing of Ansible roles provided by other products, and by the site.  Non-root user access may be blocked during node configuration by enabling the `uan-set-nologin` and `uan-unset-nologin` configuration
> layers shown in the example bootprep file below. The parameterized fields are defined in
> a `products_vars.yml` file.
> 
> **IMPORTANT** Do not remove or reorder the first three layers. The UAN product requires these layers and this specific order. Also, keep the required `cos-application-last` layer, is as the last or second to last
> layer in the configuration if `uan-set-nologin` and `uan-unset-nologin` are active.

Reviewer: ```suggestion
CFS uses configuration layers. Configuration layers allow the sharing of Ansible roles provided by other products, and by the site.  Non-root user access may be blocked during node configuration by enabling the `uan-set-nologin` and `uan-unset-nologin` configuration layers shown in the example bootprep file below. The parameterized fields are defined in a `products_vars.yml` file.

**IMPORTANT** Do not remove or reorder the first three layers. The UAN product requires these layers and this specific order. Also, keep the required `cos-application-last` layer, is as the last or second to last layer in the configuration if `uan-set-nologin` and `uan-unset-nologin` are active.
```

Configuration of UAN nodes is performed by the Configuration Framework Service \(CFS\). CFS can apply configuration to both images and nodes. When the configuration is applied to nodes, the nodes must be booted and accessible through SSH over the Node Management Network \(NMN\).

The preferred method of creating CFS configurations is to use the Shasta Admin Toolkit
(SAT) `sat bootprep` command.  This command automates the creation of IMS images, CFS
configurations, and BOS session templates. See [Create UAN Boot Images](Create_UAN_Boot_Images.md) for more details.

CFS uses configuration layers. Configuration layers allow the sharing of Ansible roles provided by other products, and by the site.  Non-root user access may be blocked during node configuration by enabling the `uan-set-nologin` and `uan-unset-nologin` configuration
layers shown in the example bootprep file below. The parameterized fields are defined in
a `products_vars.yml` file.

> **NOTE** The first three layers are required and must not be removed or reordered.
> The `cos-application-last` layer, is required and must be the last or second to last
> layer in the configuration if `uan-set-nologin` and `uan-unset-nologin` are active.

Reviewer 2: Let's give this advice more gravity and make it active voice:
```suggestion
**IMPORTANT** Do not remove or reorder the first three layers. The UAN product requires these layers and this specific order. Also, keep the required `cos-application-last` layer, is as the last or second to last
layer in the configuration if `uan-set-nologin` and `uan-unset-nologin` are active.
```

  ***NOTE:*** If a UAN layer is used in the Compute node CFS configuration, the `uan_interfaces` role will configure the default route on Compute nodes to be on the HSN, if the BICAN System Default Route is set to `CHN`.
- [`uan_motd`](uan_motd.md): this role Provides a default message of the day that can be customized by the administrator.
- [`uan_ldap`](uan_ldap.md): this optional role configures the connection to LDAP servers. To disable this role, the administrator must set 'uan_ldap_setup:no' in the 'uan-config-management' VCS repository.

> - [`uan_hardening`](uan_hardening.md): This role configures site/customer-defined network
> security of UANs, for example, preventing ssh out of the UAN over the NMN to NCN nodes.

Reviewer: ```suggestion
- [`uan_hardening`](uan_hardening.md): This role configures site/customer-defined network security of UANs, for example, preventing ssh out of the UAN over the NMN to NCN nodes.
```
